portable/BufferManagement/BufferAllocation_2.c

```diff
@@ -61,15 +59,21 @@
     #define baMINIMAL_BUFFER_SIZE    sizeof( ARPPacket_t )
 #endif /* ipconfigUSE_TCP == 1 */
 
-/*_RB_ This is too complex not to have an explanation. */
 #if defined( ipconfigETHERNET_MINIMUM_PACKET_BYTES )
-    #define ASSERT_CONCAT_( a, b )    a ## b
-    #define ASSERT_CONCAT( a, b )     ASSERT_CONCAT_( a, b )
-    #define STATIC_ASSERT( e ) \
+
+/* Crteating a static assert that can be used with sizeof()
```

Comment:
```suggestion
/* Creating a static assert that can be used with sizeof()
```

I wonder why spell-check did not catch this...

Reply:
I have now changed the text, thanks.